etl/grapher_helpers.py

```diff
@@ -151,7 +159,7 @@ def _yield_wide_table(
 
             # set new metadata with dimensions
             tab.metadata.short_name = short_name
-            tab = tab.rename(columns={column: short_name})
+            tab.rename(columns={column: short_name}, inplace=True)
```

Comment:
out of curiosity, is `inplace` improving performance? I thought i was behaving the same way.

Reply:
It is by a little bit. This is an extreme case though, I tried to heavily optimize it. Normally, I'd never use `inplace`.